### PR TITLE
cleanup remnants of innerRef and fix bubbling

### PR DESCRIPTION
**Pull request by mshwery**
## Overview 
This PR cleans up a few last remnants of innerRef and typings
 
## Screenshots (if applicable) 
NA

## Testing
- [x] Updated Typescript types and/or component PropTypes 
- [ ] Added / modified component docs 
- [ ] Added / modified Storybook stories

### CODE REVIEW

File: src/autocomplete/src/Autocomplete.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ const Autocomplete = memo(
           getRootProps,
           ...restDownshiftProps
         }) => (
-          <Pane width="100%" {...getRootProps({ refKey: 'innerRef' })}>
```
Comment (mshwery) [inline src/autocomplete/src/Autocomplete.js:165]: default is `ref` so we good here.

**Comment by cstrat** [comment]
Not the right place to ask but for the first time I've had to use refs (focus mgmt on a field) and I've had to use `innerRef` which isn't really documented anywhere that I could find... and from this update, it sounds like it is going away in favor of just using `ref`?

My example is using a `<TextInputField>` element.
Using `innerRef` **works** but I am getting proptype errors from `<Text>` and `<Box>` for `innerRef`.
Using `ref` doesn't work at all...

I'm running v5 already ("evergreen-ui": "^5.0.0-2").

**Comment by mshwery** [comment]
@cstrat can you see which version(s) of ui-box you have installed? `yarn why ui-box`

**Comment by cstrat** [comment]
My issue was I was running 5.0.0-2, updated to -5 and its working fine with no warnings and errors using `ref` 👍

**Comment by cstrat** [comment]
By the way, updating to -5 has stuffed up `<IconButton>` padding.

image

I know it is a WIP branch, but is that something you want bug reports on?

**Comment by cstrat** [comment]
https://github.com/segmentio/evergreen/issues/900

However I think I am just running an old branch! Bug doesn't appear in the later releases.

**Pull request merged**